source/extensions/filters/common/expr/context.cc

```diff
@@ -385,6 +385,8 @@ absl::optional<CelValue> XDSWrapper::operator[](CelValue key) const {
     const absl::string_view filter_chain_name =
         filter_chain_info.has_value() ? filter_chain_info->name() : absl::string_view{};
     return CelValue::CreateStringView(filter_chain_name);
+  } else if (value == Node) {
```

Comment:
I think it should be "xds.node" since we already combined them under "xds" attribute. Can you please apply back the change to move it to "xds" bag, and then it should be OK.